notebooks/heading_similarity.ipynb

```diff
@@ -0,0 +1,1668 @@
+{
```

Comment:
<p>Are all of the counts strictly smaller? (Looks that way to me but I haven't checked in detail)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/greenelab/indices/pull/8/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>

Reply:
<p>Yep, they're strictly smaller</p>